### PR TITLE
add my bio

### PR DESCRIPTION
**Pull request by krystalttle**
@githubteacher, I would like to have your :thought_balloon: on this pul request

### CODE REVIEW

**Comment by krystalttle** [comment]
@krystalttle, you too

**Comment by jyang49** [comment]
testing comment

**Comment by deeNainani** [comment]
We live in a real world, so the location should also be real :laughing:

File: docs/krystalttle.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# krystalttle bio
+
+- **Name:** krystal
+- **Occupation:** student
+- **Location:** Virtual
```
Comment (krystalttle) [inline docs/krystalttle.md:5]: I agree location should be real

File: docs/krystalttle.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# krystalttle bio
+
+- **Name:** krystal
```
Comment (krystalttle) [inline docs/krystalttle.md:3]: should name be username?

**Pull request merged**